src/cmake/shared-sources.cmake

```diff
@@ -0,0 +1,268 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Library of sources shared between app and tests
+add_library(shared-sources INTERFACE)
```

Comment:
Why do we have a shared cmake and sources.cmake as opposed to combining all in one `sources.cmake`?

Reply:
This file is shared between tests and mozillavpn. If I merge the two files I'd have to include mozillavpn-sources in the tests instead. That would require a lot of changes, mostly band aids to get it to work. I think keeping this sources file separate for now is cleaner than that.